src/EditorFeatures/CSharp/QuickInfo/SemanticQuickInfoProvider.cs

```diff
@@ -13,6 +14,31 @@ namespace Microsoft.CodeAnalysis.Editor.CSharp.QuickInfo
     [ExportQuickInfoProvider(PredefinedQuickInfoProviderNames.Semantic, LanguageNames.CSharp)]
     internal class SemanticQuickInfoProvider : AbstractSemanticQuickInfoProvider
     {
+        /// <summary>
+        /// If the token is the '=>' in a lambda, or the 'delegate' in an anonymous function,
+        /// return the syntax for the lambda or anonymous function.
+        /// </summary>
+        protected override bool GetBindableNodeForTokenIndicatingLambda(SyntaxToken token, out SyntaxNode found)
+        {
+            if (token.IsKind(SyntaxKind.EqualsGreaterThanToken)
+                && token.Parent.IsKind(SyntaxKind.ParenthesizedLambdaExpression, SyntaxKind.SimpleLambdaExpression))
+            {
+                // () =>
+            }
```

Comment:
not sure how i feel about this style of no-body then falling out to common code :)